bin/steps/appdynamics-setup

```diff
@@ -0,0 +1,76 @@
+BUILD_DIR='/app'
+echo "Setting up for Appdynamics"
+
+if ! grep -Fxq "appdynamics" /app/requirements.txt
+then
+  echo "appdynamics" >> /app/requirements.txt
```

Comment:
To Reuse the pip-install script for appdynamics agent installation. We updated requirements.txt with appdynamics. Another way is using directly pip command to separately install Agent. Let me know if i need to go ahead with second approach.